### PR TITLE
Add metadata for corpora

### PR DESCRIPTION
**Pull request by juberti**

### CODE REVIEW

File: fixieai/agents/code_shot.py [hunk 1]
```diff
@@ -25,6 +25,21 @@
 ACCEPTED_FUNC_NAMES = re.compile(r"^\w+$")
 
 
+@pydantic_dataclasses.dataclass
+class DocumentLoader:
```
Comment (hessamb) [inline fixieai/agents/code_shot.py:32]: Please put these in `corpora.py` so the docs look nicer

File: pyproject.toml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "fixieai"
-version = "0.1.10"
+version = "0.1.11"
```
Comment (hessamb) [inline pyproject.toml:3]: I'd not bump the version yet

**Comment by juberti** [comment]
PTAL

File: fixieai/agents/code_shot.py [hunk 3]
```diff
@@ -37,6 +38,7 @@ class AgentMetadata:
 
     base_prompt: str
     few_shots: List[str]
+    corpora: Optional[List[corpora.DocumentCorpus]]
```
Comment (hessamb) [inline fixieai/agents/code_shot.py:41]:
```suggestion
    corpora: Optional[List[corpora.DocumentCorpus]] = None
```

**Review by hessamb — APPROVED** [review]
LG with nits

**Pull request merged**